package.json

```diff
@@ -18,12 +18,17 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write 'src/**/*.js'",
+    "format": "prettier --write '**/*.js'",
```

Comment:
Do we really want to format files outside of `src/`? This means we format things like our cache.

<img width="435" alt="screen shot 2018-05-07 at 22 05 42" src="https://user-images.githubusercontent.com/5721314/39725164-d5173366-5242-11e8-8c2a-8f8fdcb03a30.png">